src/StudentSystemServices/Shibboleth/Controllers/ShibbolethController.php

```diff
@@ -137,6 +137,10 @@ private function getServerVariable($variableName)
 
         $variable = Request::server($variableName);
 
+        if (!empty($variable) && strtolower($variableName) === 'itrustsuppress') {
+            return strtolower($variable) === 'true';
+        }
+
         return (!empty($variable)) ?
             $variable :
             Request::server('REDIRECT_' . $variableName);
```

Comment:
@bwharrisUIC - Okay. What if `empty($variable)` is true in `f (!empty($variable) && strtolower($variableName) === 'itrustsuppress') {`?  Can we both agree that it will then slip down to the original  return of

```PHP
return (!empty($variable)) ?
            $variable :
            Request::server('REDIRECT_' . $variableName);
```
?

If you are with me so far then let me keep pulling on this scenario. What if `REDIRECT_iTrustSuppress` is located in the SERVER array and contains a value of string `'true'` or `'false'`. Do you see how we're back in a similar position as before where `getServerVariable()` is going to return a string `'true'` or `'false'`?